### PR TITLE
Add NHS.UK page examples (Issue 183)

### PR DESCRIPTION
**Pull request by mcheung-nhs**
## Description
New NHS.UK page examples

## Checklist

- [ ] Tested against our [testing policy](https://github.com/nhsuk/nhsuk-frontend/blob/master/docs/contributing/testing.md) (Resolution, Browser & Accessibility)
- [x] CHANGELOG entry

### CODE REVIEW

**Comment by Tosin-Balogun** [comment]
This is looking great, hopefully if it is well merged. We have to consider how we inform users about it

File: app/assets/sass/main.scss [hunk 1]
```diff
@@ -5,5 +5,10 @@
 @import 'components/related-nav';
 
 ///////////////////////////////////////////
-// Add your custom CSS/Sass styles below... 
+// Add your custom CSS/Sass styles below...
 ///////////////////////////////////////////
+
+// reset visited link colour
```
Comment (chrimesdev) [inline app/assets/sass/main.scss]: Is this something that we want to used across all of the prototype kit?

If so, needs new line at end of file

File: docs/assets/sass/nhsuk.scss [hunk 2]
```diff
@@ -0,0 +1,643 @@
...
+  }
+ }
+ .beta-hub-bottom-content>ul+h2 {
+   padding-top: 4px;
+ }
+ @media (min-width:40.0625em) {
+   .beta-hub-bottom-content>ul+h2 {
+     padding-top: 8px;
+   }
+ }
```
Comment (chrimesdev) [inline docs/assets/sass/nhsuk.scss]: Needs a new line at end of file

**Comment by chrimesdev** [comment]
I wonder if we need to make sure that all styles in `nhsuk.scss` are prefixed with `app-` or `beta-` because they might interfere with prototypes that are using classes from the frontend library

File: docs/assets/sass/nhsuk.scss [hunk 3]
```diff
@@ -0,0 +1,643 @@
...
+    right: 0;
+  }
+  .nhsuk-search__input {
+    border: 1px solid #ffffff;
+    height: 40px;
+  }
+}
+
+@media (min-width: 48.0625em) {
+  .nhsuk-header__navigation {
```
Comment (chrimesdev) [inline docs/assets/sass/nhsuk.scss:48]: Example of a `nhsuk-` prefix that might interfere with prototypes using the Header component, but only should be being used on the NHS website Header

File: docs/assets/sass/nhsuk.scss (hunk 2 above)
Comment (mcheung-nhs) [reply docs/assets/sass/nhsuk.scss]: Added new line

File: app/assets/sass/main.scss (hunk 1 above)
Comment (mcheung-nhs) [reply app/assets/sass/main.scss]: Moved this to `nhsuk.css` and added new line.

**Comment by chrimesdev** [comment]
Imported the colour variables into the custom nhs.uk styles so the build pipeline should pass now

**Comment by chrimesdev** [comment]
This looks good to go now from a code point of view, @karlgoldstraw is going to have a quick review then we can look at getting this merged and released.

**Comment by Tosin-Balogun** [comment]
> This looks good to go now from a code point of view, @karlgoldstraw is going to have a quick review then we can look at getting this merged and released.

It's happening!!! 🤯

**Review by karlgoldstraw — APPROVED** [review]
Wow really nice work.

**Pull request merged**